src/Kernel.php

```diff
@@ -61,13 +62,13 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
         $loader->load($confDir . '/{services}_' . $this->environment . self::CONFIG_EXTS, 'glob');
     }
 
-    protected function configureRoutes(RouteCollectionBuilder $routes): void
+    protected function configureRoutes(RoutingConfigurator $routes): void
```

Comment:
I think we can remove these method. This is now in KernelTrait since Symfony 5.4.

Reply:
You can also remove some other methods on this Kernel.